### PR TITLE
[4.0] User Menu heaader

### PR DESCRIPTION
**Pull request by brianteeman**
The display of the user name in the user dropdown menu looks a little off due to lack of styling.

This add an icon before the username and the text "signed in as " as a prefix. (The text is as used on github)

There are css changes so npm i or node buuld.js --compile-css

### Before
image

### After
image

Apologies to @chmst I think you suggested somethign like this before and I was not in favour - I was wrong.

### CODE REVIEW

File: administrator/modules/mod_user/tmpl/default.php [hunk 1]
```diff
@@ -32,7 +32,10 @@
 		<span class="fas fa-angle-down" aria-hidden="true"></span>
 	</button>
 	<div class="dropdown-menu dropdown-menu-right">
-		<div class="dropdown-header"><?php echo $user->name; ?></div>
+		<div class="dropdown-header">
+			<span class="fas fa-user-circle" aria-hidden="true"></span>
```
Comment (Quy) [inline administrator/modules/mod_user/tmpl/default.php]:
```suggestion
			<span class="fas fa-fw fa-user-circle" aria-hidden="true"></span>
```

Comment (brianteeman) [reply administrator/modules/mod_user/tmpl/default.php]: why?

Comment (Quy) [reply administrator/modules/mod_user/tmpl/default.php]: To make the icons/text line up. I assume the added CSS is to do that but I can’t tell from the screenshots.

Comment (brianteeman) [reply administrator/modules/mod_user/tmpl/default.php]: doh - my fault - I was thinking fa-fw was white not wide.

**Comment by Quy** [comment]
Is the added css still required?

**Comment by brianteeman** [comment]
yes because thats the same css as on the items below the header

File: administrator/templates/atum/scss/blocks/_header.scss [hunk 2]
```diff
@@ -205,6 +205,16 @@
       [dir=rtl] & {
         text-align: right;
       }
+
+      .fas {
+        [dir=ltr] & {
+          margin-right: $dropdown-item-padding-x;
+        }
+
+        [dire=rtl] & {
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_header.scss]:
```suggestion
        [dir=rtl] & {
```

Comment (brianteeman) [reply administrator/templates/atum/scss/blocks/_header.scss]: grrrh - too early and I need more coffee

**Comment by brianteeman** [comment]
drone failures are the usual

File: administrator/language/en-GB/mod_user.ini [hunk 3]
```diff
@@ -7,4 +7,5 @@ MOD_USER="User Menu"
 MOD_USER_ACCESSIBILITY_SETTINGS="Accessibility Settings"
 MOD_USER_EDIT_ACCOUNT="Edit Account"
 MOD_USER_MENU="User Menu"
+MOD_USER_TITLE="Signed in as"
```
Comment (SharkyKZ) [inline administrator/language/en-GB/mod_user.ini]:
```suggestion
MOD_USER_TITLE="Signed in as %s"
```

File: administrator/modules/mod_user/tmpl/default.php [hunk 4]
```diff
@@ -32,7 +32,10 @@
 		<span class="fas fa-angle-down" aria-hidden="true"></span>
 	</button>
 	<div class="dropdown-menu dropdown-menu-right">
-		<div class="dropdown-header"><?php echo $user->name; ?></div>
+		<div class="dropdown-header">
+			<span class="fas fa-fw fa-user-circle" aria-hidden="true"></span>
+			<?php echo Text::_('MOD_USER_TITLE'); ?> <?php echo $user->name; ?>
```
Comment (SharkyKZ) [inline administrator/modules/mod_user/tmpl/default.php]:
```suggestion
			<?php echo Text::sprintf('MOD_USER_TITLE', $user->name); ?>
```

**Review by SharkyKZ — CHANGES_REQUESTED** [review]
Please use variable in language string.

**Comment by brianteeman** [comment]
@SharkyKZ thanks - wasn't thinking correctly this morning obviously

File: administrator/language/en-GB/mod_user.ini (hunk 3 above)
Comment (Quy) [reply administrator/language/en-GB/mod_user.ini]: @brianteeman Commit this too.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5f0b3234a927512bba7195667829be22ce63c30c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27893">issues.joomla.org/tracker/joomla-cms/27893</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 5f0b3234a927512bba7195667829be22ce63c30c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27893">issues.joomla.org/tracker/joomla-cms/27893</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27893">issues.joomla.org/tracker/joomla-cms/27893</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**